Preview text (#740)
pulumi/pulumi-cloud: README.md

[![Build Status](https://travis-ci.com/pulumi/pulumi-cloud.svg?token=eHg7Zp5zdDDJfTjY8ejq&branch=master)](https://travis-ci.com/pulumi/pulumi-cloud)

# Pulumi Cloud Framework (Preview)

> Pulumi's multi-cloud framework for building modern container and serverless cloud applications.

Reviewer: ```suggestion
Pulumi Cloud Framework is our multi-cloud framework for building modern container and serverless cloud applications.
```

> This is a preview library demonstrating how to build Pulumi [component

Reviewer: ```suggestion
This is a preview library demonstrating how to build Pulumi [components]
```

> resources](https://www.pulumi.com/docs/reference/component-tutorial/) that can abstract over and

Reviewer: ```suggestion
(https://www.pulumi.com/docs/reference/component-tutorial/) that can abstract over and
```

target multiple clouds.

For developers targeting a single cloud platform like AWS, Azure or GCP, we recommend using the
[@pulumi/aws](https://github.com/pulumi/pulumi-aws),
[@pulumi/azure](https://github.com/pulumi/pulumi-azure) and

> [@pulumi/gcp](https://github.com/pulumi/pulumi-gcp) packages respectively.  These packages give full

Reviewer: If we go with my suggestion, I'll just try to make it consistent here.

> A platform-specific package gives you full access to the breadth of its capabilities and comes with many abstractions to make development easier.
